PluginCore/PluginCore/Resources/zh_CN.resx

```diff
@@ -6112,4 +6112,8 @@ AND - All conditions are met.</value>
     <value>Defines which method should be used to clean the contents of the output panel.</value>
     <comment>Added after 5.2.0</comment>
   </data>
+  <data name="HaXeContext.Description.CompletionServerWaitStdio" xml:space="preserve">
+    <value>Makes haxe completion server use stdin and stderr instead of sockets for communication with IDE. (only applies to CompletionServer Completion Mode).</value>
```

Comment:
Not that my English is perfect, but I'd say:

Makes Haxe completion server to use stdin and stderr instead of sockets for communication with the IDE (only applies to CompletionServer Completion Mode and Haxe 3.3+).